backend/app/controllers/users.js

```diff
@@ -1171,3 +1178,33 @@ function* insertOne(req, res, next) {
     }
     return user;
 }
+
+function _createUserOnAuthService(email, password, roleId) {
```

Comment:
@Perry5 are we going to need some sort of callback on this? Either for calling it internally or for potentially using a promise with the created user?

Reply:
Thats a good point. I'll go ahead and make this a callback